gnovm/memfile.go

```diff
@@ -10,6 +10,7 @@ import (
 type MemFile struct {
 	Name string `json:"name" yaml:"name"`
 	Body string `json:"body" yaml:"body"`
+	Path string `json:"path" yaml:"path"`
```

Comment:
It's best if it's not in the MemFile directly, as MemFiles are part of the import paths. Instead, cmd/gno should have a mechanism to resolve import path + filename to a filename